src/style.css

```
@@ -1 +1,21 @@
 /* styles go here */
+
+html {
+  box-sizing: border-box;
+  padding: 0;
+  margin-top: 0;
+}
+
+form {
+  margin: 0;
+}
+
+.form-fild {
+  display: block;
+  margin: 9px 0;
+}
+
+.form-fieldset {
+  padding: 0 12px;
+  margin-bottom: 20px;
+}
```

```suggestion
.form-field {
  display: block;
  margin-bottom: 10px;
}

.fieldset {
  margin-bottom: 20px;
}

.fieldset > .form-field:last-of-type {
  margin-bottom: 0;
}
```

спробуй такий варіант

ніяких стилів, крім нижніх марджинів непотрібно